src/legacy/smartweave-global.ts

```diff
@@ -260,6 +280,9 @@ export class SWVrf {
 
   // returns a random value in a range from 1 to maxValue
   randomInt(maxValue: number): number {
```

Comment:
please mark as Deprecated and update the docs here https://docs.warp.cc/docs/sdk/advanced/vrf

Reply:
I have added comments with `@deprecated`. I will update all documentation related to the new sequencer as part of the task at https://github.com/warp-contracts/sequencer/issues/85.